### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by theresakhoo**
Added some initial tests.

### CODE REVIEW

File: tests/files/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name" translatable="false">TachiyomiJ2K</string>
+    <string name="app_short_name" translatable="false">TachiJ2K</string>
+    <string name="str1">Winter is coming</string>
```
Comment (diegone) [inline tests/files/values/strings.xml:5]: It still doesn't get old? 🙄

**Review by diegone — APPROVED** [review]
Exciting to get some open source contribution. Thanks so much! 😁

File: tests/files/values/strings.xml (hunk 1 above)
Comment (theresakhoo) [reply tests/files/values/strings.xml:5]: Never!

File: tests/translators/piglatinizer.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import * as piggyTranslator from '../../translators/piglatinizer';
+
+describe ('pig latinizer translator tests', () =>{
+
+  const job = {
+    "tus": [{
```
Comment (theresakhoo) [inline tests/translators/piglatinizer.test.js:6]: BTW, in the filter, you use `translationUnits`, but in the translators, you use `tus`.  Do you want to be consistent?

Comment (diegone) [reply tests/translators/piglatinizer.test.js:6]: yeah, it's one of those inner conflicts between being concise and descriptive. Feel free to refactor!

Comment (theresakhoo) [reply tests/translators/piglatinizer.test.js:6]: Fair enough.  Was just trying to understand intent.

**Pull request merged**